Fixes to IPSO object descriptions and units

akeranen opened the pull request:
See commits for details

akeranen commented on 3333.xml:
```diff
@@ -39,7 +39,7 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
                 <RangeEnumeration>0..1</RangeEnumeration>
                 <Units>s</Units>
-                <Description>For shorter times of a fraction of a second (i.e. 0.23).</Description>
+                <Description>Fractional part of the time when sub-second precision is used.</Description>
```
Maybe we want to keep an example for clarity? For example: `(e.g., 0.23 for 230 ms)`

akeranen replied:
Added the example as discussed in the IPSO call on Oct 24th.

akeranen commented:
The more substantial changes that are worth a closer look include:

* Removing UCUM examples: https://github.com/OpenMobileAlliance/lwm2m-registry/pull/256/commits/ec76ce33fbcf17896da9ff8dd8ddf4fdb96e8762
* Simplified set-point description: https://github.com/OpenMobileAlliance/lwm2m-registry/pull/256/commits/711f9f4e076678c6a2d5210d81eaae2e6ba2fb58
* Using "write" instead of "POST": https://github.com/OpenMobileAlliance/lwm2m-registry/pull/256/commits/da5e222fa989248419b7003882bc226ba1d4033a
* Fix to fractional time description: https://github.com/OpenMobileAlliance/lwm2m-registry/pull/256/commits/e7465cfde01d5950fc01de836c36f6fff9968ca4 (also with added example https://github.com/OpenMobileAlliance/lwm2m-registry/pull/256/commits/0d8323e4767839b063cd27eb2de7da20add50e49)

akeranen commented:
The commit dbd1ab90f1aa452636363171461e6c240d885279 also fixes #233

jaudiger commented:
Hi @akeranen,

I tried to do my best to check all the changes, and be sure if no human errors have been done.

My comments:
- Since you modified the description of resource 5502 in 3200.xml and Common.xml. Is it possible to update the description of this resource for 3401.xml ?
- I saw you also updated the description of the resource 5701. When I was checking I saw there was on issue with the description of this resource for the following objects: 10254 and 10258.
- You updated the description of the Object 3305 in 3305.xml, but the file DDF.xml contains also the typo "Il also" instead of "It also".
- Some objects continue to use "-" instead of ".." for range enumeration. For IPSO objects this is true for only 10265, after that this is mainly used inside LwM2M Core objects. Not sure if we have to update those.

Jérémy

akeranen commented:
Thank you for the review @jaudiger! Good catch with 3305 description in DDF.xml; fixed now in the latest commit (along with a few others).

I agree that also objects 3401, 10254, 10258, and 10265 should be updated but those are not originally from IPSO (they are registered by other orgs) so I think we need to check with the registering orgs before doing the change. Perhaps we should track that in another issue.

jaudiger commented:
You are right, I didn't notice these objects were not from IPSO. Tracking the changes in another issue seems to be a good idea !

jpradocueva commented:
@akeranen the validation indicates the following issues with this PR:

- [ ] RR 4000, I believe that the description is missing one closing "]" as a result RR 4000 and 4001 show errors.
- [ ] Other errors, some may have not been introduced by your PRs but it is difficult to say due to all the changes that have been done so far.


Error | Time | Description | ObjectID and/or ResourceID
-- | -- | -- | --
463 |   | RangeEnumeration is different | 3200:5504
463 |   | RangeEnumeration is different | 3203:5650
463 |   | RangeEnumeration is different | 10266:6000
463 |   | RangeEnumeration is different | 10266:6001
463 |   | RangeEnumeration is different | 10267:6000
463 |   | RangeEnumeration is different | 10267:6001
463 |   | RangeEnumeration is different | 10268:6000
463 |   | RangeEnumeration is different | 10268:6001
463 |   | RangeEnumeration is different | 10269:6000
463 |   | RangeEnumeration is different | 10269:6001
463 |   | RangeEnumeration is different | 10270:6000
463 |   | RangeEnumeration is different | 10270:6001
463 |   | RangeEnumeration is different | 10271:6000
463 |   | RangeEnumeration is different | 10271:6001
463 |   | RangeEnumeration is different | 10272:6019
463 |   | RangeEnumeration is different | 10272:6024
463 |   | RangeEnumeration is different | 10273:6019
463 |   | RangeEnumeration is different | 10273:6024
463 |   | RangeEnumeration is different | 10274:6019
463 |   | RangeEnumeration is different | 10274:6024
463 |   | RangeEnumeration is different | 10275:6019
463 |   | RangeEnumeration is different | 10275:6024
463 |   | RangeEnumeration is different | 10276:6019
463 |   | RangeEnumeration is different | 10276:6024
463 |   | RangeEnumeration is different | 10277:6019
463 |   | RangeEnumeration is different | 10277:6024
463 |   | RangeEnumeration is different | 10278:6019
463 |   | RangeEnumeration is different | 10278:6024
463 |   | RangeEnumeration is different | 10279:6019
463 |   | RangeEnumeration is different | 10279:6024
463 |   | RangeEnumeration is different | 10280:6019
463 |   | RangeEnumeration is different | 10280:6024
463 |   | RangeEnumeration is different | 10281:6019
463 |   | RangeEnumeration is different | 10281:6024
463 |   | RangeEnumeration is different | 10282:6019
463 |   | RangeEnumeration is different | 10282:6024
463 |   | RangeEnumeration is different | 10283:6019
463 |   | RangeEnumeration is different | 10283:6024
463 |   | RangeEnumeration is different | 10284:6019
463 |   | RangeEnumeration is different | 10284:6024
463 |   | RangeEnumeration is different | 10290:6000
463 |   | RangeEnumeration is different | 10290:6001
463 |   | RangeEnumeration is different | 10291:6000
463 |   | RangeEnumeration is different | 10291:6001
463 |   | RangeEnumeration is different | 10291:6040
463 |   | RangeEnumeration is different | 10292:6000
463 |   | RangeEnumeration is different | 10292:6001
463 |   | RangeEnumeration is different | 10292:6040

akeranen commented:
Thanks @jpradocueva! I fixed the error in RR4000 in the latest commit. I did also make that error for objects 10254 and 10258. Fixed now in the previous commit. We should probably get rid of that CDATA trick altogether and escape unsafe characters instead.

I also fixed the range in the IPSO objects 3200 and 3203. Since the other objects (102??) are not original IPSO objects I did not touch them; and fixing them with sed would result in the EOL issues discussed in the email thread.

akeranen commented:
Here's how the range fix to the 102?? objects would look like: https://github.com/akeranen/lwm2m-registry/pull/5/files
